nano/node/network.cpp

```diff
@@ -20,7 +25,7 @@ nano::network::network (nano::node & node_a, uint16_t port_a) :
 	node (node_a),
 	publish_filter (256 * 1024),
 	udp_channels (node_a, port_a),
```

Comment:
Why not passing it also to udp_channels, so you don't need to rely on node's dependency?